src/app/app.component.ts

```diff
@@ -1,14 +1,29 @@
-import { Component} from '@angular/core';
+import { Component, HostListener, OnInit} from '@angular/core';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.scss']
 })
 
-export class AppComponent {
-  title = 'out-of-school';
-  constructor () {}
+export class AppComponent implements OnInit{
+  constructor() { }
+
+  MobileView: boolean = false;
```

Comment:
this needs to be removed from header component and be passed as input parameter from app component into header conponent

Reply:
fixed, did same thing for sidenav component

Reply:
i dont see any changes